<!DOCTYPE HTML>
<!--
	Editorial by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Portfolio by Temitope 0. Benson</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
	</head>
	<body class="is-preload">

		<!-- Wrapper -->
			<div id="wrapper">

				<!-- Main -->
					<div id="main">
						<div class="inner">

							<!-- Header -->
								<header id="header">
									<a class="logo"><strong>Portfolio</strong> by Temitope O. Benson</a>
									<ul class="icons">
										<li><a href="https://www.linkedin.com/in/temitopebenson/" class="icon brands fa-linkedin"><span class="label">LinkedIn</span></a></li>
										<li><a href="https://github.com/temitope-benson" class="icon brands fa-github"><span class="label">Github</span></a></li>
										<li><a href="mailto:temitopebenson@yahoo.com" class="icon solid fa-envelope"><span class="label">Email</span></a></li>
									</ul>
								</header>

							<!-- Banner -->
								<section id="banner">
									<div class="content">
										<header>
											<h1>Temitope O. Benson</h1>
											<p>Portfolio and Project Reports</p>
										</header>
										<div class="row">
											<div class="col-4 col-12-small">
												<span class="image object">
													<img  src="images/temitope.jpeg" alt=""  style="max-width: 300px;"/>
												</span>
											</div>
											<div class="col-8 col-12-small">
												<p>My training in Mathematics, Computer Science, and Biology allows me to investigate questions at the cutting edge of computational biology. I work on both biological experiments and modeling projects, and this allows for a high level of interdisciplinary collaboration leading to novel results. I have experience with computational genomics, bioinformatics, all types of mathematical modeling across many different programming langauges.</p>
												<ul class="actions">
													<li><a href="VirtualCV.html" class="button">Learn more about my past experience</a></li>
												</ul>
												<p>I have been building computational models of various biological systems for 10 years and have defended three of these projects in front of senior committees at different levels. I have two first author publications already and I currently am preparing my third first-author publication. I have been selected to give talks at local conferences and I have been invited to speak at international events including the Society for Mathematical Biology annual conference in 2023. </p>
												<p>I am currently looking for a new position where I can expand my knowledge and provide valuable insight. I am interested in building and working on projects including: larger AI/ML/mathematical models, bioinformatics, or computational genomics. My background makes me a unique candidate and if you have a position you think would benefit from someone who can work as both a computer scientist and a biologist, I'd love to talk to you about it.</p>
												
											</div>
											
											
										</div>
										<!-- <ul class="actions">
											<li><a href="#" class="button big">Learn More</a></li>
										</ul> -->
									</div>
									
								</section>

							<!-- Section -->
								<section>
									<header class="major">
										<h2>Selected Skills</h2>
									</header>
									<div class="features">
										<article>
											<span class="icon solid fa-code"></span>
											<div class="content">
												<h3>Programming</h3>
												<p> Python (Pandas, Numpy, SciKit-Learn, Pytorch, Keras, Tensorflow), Matlab (Parallel Computing, Statistics, Machine Learning & Deep Learning), R, GitHub, LaTeX, Parameterization, OOP, Debugging </p>
											</div>
										</article>
										<article>
											<span class="icon solid fa-microscope"></span>
											<div class="content">
												<h3>Computational Biology/Laboratory</h3>
												<p>Experimental Design & Collection, Scientific Writing (Grants, Publications), Agent-Based Modeling, Data Analysis</p>
											</div>
										</article>
										<article>
											<span class="icon solid fa-handshake"></span>
											<div class="content">
												<h3>Leadership</h3>
												<p>Management, Collaboration (with bench and computational scientists), Public Speaking, Training</p>
											</div>
										</article>
										<article>
											<span class="icon solid fa-bicycle"></span>
											<div class="content">
												<h3>Personal</h3>
												<p>Playing Soccer, Watching Movie. </br>
													Continent Hopper </p>
											</div>
										</article>
									</div>
								</section>

							<!-- Section -->
								<section>
									<header class="major">
										<h2>Projects</h2>
									</header>
									<div class="posts">
										<article>
											<a href="Centrosome.html" class="image"><img src="images/Centrosome.jpg" alt="" /></a>
											<h3>Mathematical and Biological Exploration of Centrosome Maturation</h3>
											<p><b>Skills:</b> Fluorescence Microscopy, <i>C. elegans</i> Maintenance, Data Analysis, Incorporating Data in Models, Scientific Writing & Presentations, LATEX, XMGrace, Inkscape</br><b>Programming:</b> MATLAB, ODE/PDE Models, Parallel Computing Toolbox</p>
											<p>Collecting <i>in vivo</i> data of the recruitment and recovery of a GFP tagged centrosomal protein in the early <i>C. elegans</i> embryo, I developed and parametrized a mathematical model of centrosome maturation. 
												Analysis of a large set of parameters that fit our model to the <i>in vivo</i> data reveal that there are potential differences in the dynamics between the two centrosomes. 
												The multi-scale microscopy and mathematical modeling together support our hypothesis that there is a previously uncharacterized asymmetry inside of the <i>C. elegans</i> centrosome that could be connected to cell fate decisions.</p>
											<ul class="actions">
												<li><a href="Centrosome.html" class="button">More Info</a></li>
												<li><a class="button">Publication in Progress</a></li>
											</ul>
										</article>
										<article>
											<a href="https://github.com/Shayne-Falco/Crossover-Model" class="image"><img src="images/SC Schematic.png" alt="" /></a>
											<h3>Turing Model of Crossover Positioning</h3>
											<p><b>Skills:</b> Iterative Model Design, Matlab, Scientific Writing, Multi-lab Collaboration</p>
											<p>During meiosis crossover of homologous chromosomes is required for proper development. The positioning of crossovers in <i>C. elegans</i> is a one-and-only-one process meaning a crossover will happen near either but not on the ends of the SC.
												This process has been previously hypothesized to follow the dynamics of Turing reaction-diffusion models given the observed crossover assurance and interference. I set out to recapitulate the patterning of crossovers with a reaction diffusion model.
												This shows that there could exist a mechanism that mimics the patterning of the Turing reaction-diffusion model to position a single crossover on the arms of the SC. Further, our model indicates that such a mechanism would rely strongly on the initial DSB pattern.</p>
												<ul class="actions">
													<!-- <li><a href="CrossoverPatterning.html" class="button">More Info</a></li> -->
													<li><a href = "https://github.com/Shayne-Falco/Crossover-Model" class = "button">Github & Writeup</a></li>
												</ul>
										</article>
										<article>
											<a href="https://www.sciencedirect.com/science/article/abs/pii/S0010482516301585?via%3Dihub" class="image"><img src="images/MicroCalcif.jpg" alt="" /></a>
											<h3>Agent-Based Model of Microcalcification Growth in Simulated Tissue</h3>
											<p><b>Skills:</b> JAVA, Agent-Based Model, Image Analysis</p>
											<p>When screening for breast cancer, the radiological interpretation of mammograms is a difficult task, particularly when classifying precancerous growth such as microcalcifications (MCs). 
												We found that the fractal dimension of MCs changes with breast tissue density, which suggests tissue environment plays a role in regulating MC growth. 
												Benign and malignant MCs are distinguishable in all types of tissue by shape, size, and area, which is consistent with findings in the literature. </br>
												These results may help to better understand the effects of the tissue environment on tumor progression, and improve classification of MCs in mammograms via computer-aided diagnosis.</p>
											<ul class="actions">
												<li><a href="https://www.sciencedirect.com/science/article/abs/pii/S0010482516301585?via%3Dihub" class="button">Publication</a></li>
											</ul>
										</article>
										<article>
											<a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1006800" class="image"><img src="images/pollen3D.png" alt="" /></a>
											<h3>Mathematical Modelling of Pollen Aperture Formation</h3>
											<p><b>Skills:</b> FLEXPDE, MATLAB, Reaction-Diffusion, Collaboration with Biologists, Data Visualization</p>
											<p>This work demonstrates the ability of the theoretical model to help focus experimental efforts and to provide fundamental insights into an important biological process. 
												Pollen provides an excellent system to study pattern formation at the single-cell level. 
												To explore possible mechanisms responsible for aperture patterning, we developed a mathematical model based on the Gierer-Meinhardt system of equations. 
												This model was able to recapitulate aperture patterns observed in the wild-type and higher-ploidy pollen.</p>
											<ul class="actions">
												<li><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1006800" class="button">Publication</a></li>
											</ul>
										</article>
										<article>
											<a href="Network.html" class="image"><img src="images/Network.png" alt="" /></a>
											<h3>Mathematical Modelling of the Centrosome Recruitment Pathway</h3>
											<p><b>Skills:</b> Julia, Literature Review, Signaling Networks</p>
											<p>Using the literature, I built a mathematical model of the centrosome recruitment pathway. I will use this model to explore the effects of different perturbations on the pathway. I am interested in exploring dynamics that are required for the proper recruitment of centrosome factors. </p>
											<ul class="actions">
												<li><a href="Network.html" class="button">More Info</a></li>
											</ul>
										</article>
										<article>
											<a href="Centrosome_data_analysis.html" class="image"><img src="images/Centrosome.jpg" alt="" /></a>
											<h3>Centrosome Maturation Parameters Analysis</h3>
											<p><b>Skills:</b> Pandas, Regression, Data Visualization</p>
											<p>Exploratory data analysis on the parameter sets discovered fitting the ODE model to the <i>in vivo</i> centrosome recruitment and recovery data. I was answering the question of if there were correlated parameters from the parameter search and if the random search used was critical to the parameters produced. I found the random search did not matter and that there were key correlations that could describe the mechanisms for centrosome factor recruitment.</p>
											<ul class="actions">
												<li><a href="Centrosome_data_analysis.html" class="button">More Info</a></li>
											</ul>
										</article>
										<article>
											<a class="image"><img src="images/DissertationTitle.png" alt="" /></a>
											<h3>PhD Dissertation: Combining Biological Experimentation with Precise Quantification and Mathematical Modeling</h3>
											<p><b>Skills:</b>Yes</p>
											<p>The complexity of biological systems often defies straightforward understanding due to the multitude of interacting components and intricate feedback loops. 
											 
												Mathematical models can serve as abstract representations of real-world phenomena, capturing essential features while stripping away extraneous details. 
												By formulating biological processes in mathematical terms, I gain the ability to simulate, manipulate, and explore the behavior of these systems under various conditions. 
												Reaction-diffusion models and kinetic models are used to build a better understanding of otherwise difficult or impossible to determine biological processes. 
												Combining novel mathematical modeling and a precise quantification of biological processes can uncover previously unseen mechanisms. 
												
											</br></br>In this dissertation I explore three different systems, <i>C. elegans</i> centrosome maturation, <i>C. elegans</i> crossovers during meiosis, and <i>Hydra vulgaris</i> cell aggregate regeneration. 
												In all of these models I iterate from a simple explanation of the biological system by adding new dynamics and complexity just until our models are able to recapitulate the data. 
												This ethos allows for relatively simple models where all of the dynamics have a strong biological understanding. 
												By taking an unbiased approach to experimental design and analysis I was able to simulate biological processes with many different types of models. </p>
											
										</article>
										<article>
											<a href="NLP.html" class="image"><img src="images/meter-true.jpg" alt="" /></a>
											<h3>Fake News Prediction Model</h3>
											<p><b>Skills:</b> NLP, Pandas, Python, Data Analysis, Machine Learning</p>
											<p>This work demonstrates the ability of the models to detect fake news when the source of the news is unknown. Previous data sets on Kaggle and the models developed from them have a very high accuracy because the source of the news is so highly correlated with the truthfulness. Our team developed a model that could predict the truthfulness of news by analyzing synthetic features that we extracted from the text of the news article.</p>
											<ul class="actions">
												<li><a href="NLP.html" class="button">More Info</a></li>
												<li><a href = "https://www.erdosinstitute.org/certificates/spring-2023/data-science-boot-camp/shayne-plourde" class = "button">Certificate of Completion</a></li>
											</ul>
										</article>
										
										<!--<article>
											<a href="#" class="image"><img src="images/pic04.jpg" alt="" /></a>
											<h3>Future Personal Projects</h3>
											<p>Aenean ornare velit lacus, ac varius enim lorem ullamcorper dolore. Proin aliquam facilisis ante interdum. Sed nulla amet lorem feugiat tempus aliquam.</p>
											<ul class="actions">
												<li><a href="#" class="button">More</a></li>
											</ul>
										</article>-->
									</div>
								</section>

						</div>
					</div>

				<!-- Sidebar -->
					<div id="sidebar">
						<div class="inner">

							

							<!-- Menu -->
								<nav id="menu">
									<header class="major">
										<h2>Menu</h2>
									</header>
									<ul>
										<li><a href="index.html">Homepage</a></li>
										<li>
											<span class="opener">Centrosome Project Reports</span>
											<ul>
												<li><a href="Centrosome.html">Centrosome Maturation</a></li>
												<li><a href="Centrosome_data_analysis.html">Parameter Analysis</a></li>
												<li><a href="Network.html">Network Model</a></li>
											</ul>
										</li>
										<li><a href="NLP.html">Fake News Detection</a></li>
										<li><a href="VirtualCV.html">Experience</a></li>
									</ul>
								</nav>

							<!-- Section 
								<section>
									<header class="major">
										<h2>Ante interdum</h2>
									</header>
									<div class="mini-posts">
										<article>
											<a href="#" class="image"><img src="images/pic07.jpg" alt="" /></a>
											<p>Aenean ornare velit lacus, ac varius enim lorem ullamcorper dolore aliquam.</p>
										</article>
										<article>
											<a href="#" class="image"><img src="images/pic08.jpg" alt="" /></a>
											<p>Aenean ornare velit lacus, ac varius enim lorem ullamcorper dolore aliquam.</p>
										</article>
										<article>
											<a href="#" class="image"><img src="images/pic09.jpg" alt="" /></a>
											<p>Aenean ornare velit lacus, ac varius enim lorem ullamcorper dolore aliquam.</p>
										</article>
									</div>
									<ul class="actions">
										<li><a href="#" class="button">More</a></li>
									</ul>
								</section>-->

							<!-- Section -->
								<section>
									<header class="major">
										<h2>Get in touch</h2>
									</header>
									
									<ul class="contact">
										<li class="icon solid fa-envelope"><a href="mailto:shayne.plourde@outlook.com">shayne.plourde@outlook.com</a></li>
										<li class="icon brands fa-linkedin"><a href="https://www.linkedin.com/in/shayne-plourde/">www.linkedin.com/in/shayne-plourde/</a></li>
										<li class="icon brands fa-github"><a href="https://github.com/Shayne-Falco">github.com/Shayne-Falco</a></li>
									</ul>
								</section>

							<!-- Footer -->
								<footer id="footer">
									<p class="copyright">&copy; Untitled. All rights reserved. Demo Images: <a href="https://unsplash.com">Unsplash</a>. Design: <a href="https://html5up.net">HTML5 UP</a>.</p>
								</footer>

						</div>
					</div>

			</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>
